src/fraudnet.js

```diff
@@ -91,7 +88,6 @@ export const createFraudnetScript = ({
 
     fraudnetScript.setAttribute("nonce", cspNonce || "");
     fraudnetScript.setAttribute("src", fraudnetUrl);
-    fraudnetScript.addEventListener("error", () => resolve());
```

Comment:
wouldn't we still want this? in case there's an error loading the script, otherwise it would stall out forever?

Reply:
You are correct, thank you. I think this snuck in with a merge conflict or something. I'll update.

Reply:
Oh oops nevermind it's actually still there but this one was a duplicate to one below where we actually handle the error within this fn